hiera/hiera.go

```diff
@@ -112,6 +114,58 @@ func Lookup2(
 	return nil
 }
 
+// MultiLookup performs a lookup using the given parameters for all of the names passed in.
+//
+// ic - The lookup invocation
+//
+// names[] - The name or names to lookup
+//
+// valueType - Optional expected type of the found value
+//
+// defaultValue - Optional value to use as default when no value is found
+//
+// override - Optional map to use as override. Values found here are returned immediately (no merge)
+//
+// defaultValuesHash - Optional map to use as the last resort (but before defaultValue)
+//
+// options - Optional map with merge strategy and options
+//
+// defaultFunc - Optional function to produce a default value
+func MultiLookup(
+	ic api.Invocation,
+	names []string,
+	valueType dgo.Type,
+	defaultValue dgo.Value,
```

Comment:
Not sure if `defaultValue` and `defaultFunc` makes sense here. Default for what key? The `defaultValuesHash` is enough.